municipal_finance/models/updates.py

```diff
@@ -0,0 +1,55 @@
+
+from django.db import models
+
+from django.contrib.auth.models import User
+
+
+class BaseUpdate(models.Model):
+    id = models.AutoField(primary_key=True)
+    user = models.ForeignKey(User, models.DO_NOTHING)
+    datetime = models.DateTimeField(auto_now_add=True)
+    deleted = models.BigIntegerField(null=True)
+    inserted = models.BigIntegerField(null=True)
+
+    class Meta:
+        abstract = True
+
+
+class MunicipalStaffContactsUpdate(BaseUpdate):
+    file = models.FileField(
```

Comment:
it might be a good idea to add a timestamp or uuid to the file name to ensure it's unique and that we don't overwrite one file with another and then get confused when we're debugging something.

https://github.com/vulekamali/datamanager/blob/master/budgetportal/models.py#L1864

I'm happy for that to be added to the backlog though and merging this as is

Reply:
@jbothma Implemented using timestamps for file names.